setup

```diff
@@ -17,7 +17,7 @@ if test "$(uname)" = "Darwin"; then
     brew upgrade ansible
   fi
 elif test "$(uname)" = "Linux"; then
-  if test -f /etc/lsb-release -a ! "$(which ansible)"; then
+  if test -f /etc/lsb-release && -a ! "$(which ansible)"; then
```

Comment:
I didn't understand wha't you'r trying to do here.

The `-a` is already an `AND`, the `&&` before it will actually break the `if` clause...